### PR TITLE
string permissions

### PR DESCRIPTION
**Pull request by EliasEriksson**
* api responds and accepts string permissions
* removed the ability to add users with `POST /organization` users now have to be added with `PATCH /organization`
* updated readme to cover the new way to initialize users
* updated readme with updated email configuration for cloudflare workers that happened in the summer

### CODE REVIEW

File: Context/Users/Storage/Context/Users/User.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import { cryptly } from "cryptly"
+import { flagly } from "flagly"
 import { isoly } from "isoly"
 import { userwidgets } from "@userwidgets/model"
 import { isly } from "isly"
+import type { Users } from "."
```
Comment (EliasEriksson) [inline Context/Users/Storage/Context/Users/User.ts:6]: speficy index

File: Context/Users/index.ts [hunk 2]
```diff
@@ -149,7 +162,8 @@ export class Users {
 		if (gracely.Error.is(updated))
 			result = updated
 		else {
-			if (user.permissions)
+			// changed from user.permissions. it was mutated. this must be correct?
```
Comment (EliasEriksson) [inline Context/Users/index.ts:165]: this is tested. this is correct. remove comment

**Review by EliasEriksson — COMMENTED** [review]
fix this

**Review by linusarnejohansson — APPROVED** [review]
looks good to me

**Review by TheLillyleaf — APPROVED** [review]

**Pull request merged**